cypress/fixtures/pages/DmView/useAdventure/68c8bd92-04ff-4359-9856-8d2d6b02b69b.json

```diff
@@ -0,0 +1,139 @@
+{
+  "id": "68c8bd92-04ff-4359-9856-8d2d6b02b69b",
```

Comment:
What are these JSON files with hashes for names?

Reply:
Oh! If you look at the test file, you'll see code like this:

```
 cy.intercept('GET', `${TEST_API_BASE}/adventure/${TEST_ADVENTURE_ID}`, {
      statusCode: 200,
      fixture: `pages/DmView/useAdventure/${TEST_ADVENTURE_ID}`
    });
```

The string value for fixture key is just a path to this JSON file. I'm just using the id of the record as the file name, but we could use anything and call this whatever.